data_pipeline/twitter_ads_api/twitter_ads_api_pipeline.py

```diff
@@ -68,37 +69,84 @@ def get_bq_compatible_json_response_from_resource_with_provenance(
 
 
 def get_param_dict_from_api_query_parameters(
-    api_query_parameters: TwitterAdsApiApiQueryParametersConfig,
+    api_query_parameters_config: TwitterAdsApiApiQueryParametersConfig,
     value_from_bq: str,
     start_time: str,
-    end_time: str
+    end_time: str,
+    placement: Optional[str] = None
 ) -> dict:
+    if placement:
```

Comment:
maybe this could be:

```python
    result = {
        api_query_parameters_config.parameter_names_for.bigquery_value: value_from_bq,
        api_query_parameters_config.parameter_names_for.start_time: start_time,
        api_query_parameters_config.parameter_names_for.end_time: end_time
    }
    if placement:
        result[api_query_parameters_config.parameter_names_for.placement] = placement
    return result
```